src/core/components/googleButton.component.tsx

```diff
@@ -1,17 +1,42 @@
-import { withStyles } from "@material-ui/core/styles"
-import { Button } from "@material-ui/core"
+import React from "react"
+import { makeStyles, Typography, ButtonProps, withStyles, Button } from "@material-ui/core"
 import { grey } from "@material-ui/core/colors"
+import googleIcon from "../../assets/images/icon/google-icon.svg"
 
-const GoogleButtonComponent = withStyles({
+const useStyle = makeStyles(theme => ({
+  googleIcon: {
+    marginRight: theme.spacing(3.5)
+  },
+  capitalText: {
+    textTransform: "capitalize"
+  }
+}))
+
+const GoogleButtonComponent: React.FC<ButtonProps> = props => {
+  const classes = useStyle()
+  return (
+    <Button {...props}>
+      <img src={googleIcon} alt="" className={classes.googleIcon} />
```

Comment:
```suggestion
      <img src={googleIcon} alt="facebookIcon" className={classes.googleIcon} />
```

empty alt will affect to SEO